src/nile/commands/node.py

```diff
@@ -0,0 +1,7 @@
+"""Command to start StarkNet local network."""
+import subprocess
+
+
+def node_command():
+    """Start StarkNet local network."""
+    subprocess.check_call("starknet-devnet")
```

Comment:
Should it catch `FileNotFoundError` and recommend to install `starknet-devnet` using `nile install`?